README.md

```diff
@@ -1,11 +1,59 @@
-# droidboot_device_volla_yggdrasil
-This repo contains lk based bootlader for the Volla Phone. It is based on sources from BSP. Created for dualbooting. It can display a simple GUI with lvgl, and read configs from an ext2 formated cache partion.
-First entry will call boot_linux_from_storage. 
+# Android boot manager for Volla phone (droidboot_device_volla_yggdrasil)
```

Comment:
Please capitalize B and M of Android Boot Manager

Reply:
Done (I have also capitalized the P of Phone since this also done on the Volla Phone website)